packages/commonwealth/server/migrations/20221213192007-add-projects.js

```diff
@@ -51,7 +51,7 @@ module.exports = {
             default_symbol: 'CWP',
             name: 'Common Protocol',
             type: 'dao',
-            network: 'common-protocol',
+            network: 'ethereum',
```

Comment:
This change will affect Chain Events handling -- we should instead add a new loading case for common protocol.

Reply:
We could also remove it?